index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Jim Ntare | Software Engineer</title>
+    <link rel="stylesheet" href="./build/styles.css" />
+  </head>
+  <body>
+    <header class="nav_bar">
+      <div class="logo">Jim Ntare.</div>
+      <div class="nav_items">
+        <div class="nav_item">Portfolio</div>
+        <div class="nav_item">About</div>
+        <div class="nav_item">Contact</div>
+      </div>
+    </header>
```

Comment:
- [ ] This section should contain a log and Hamberger aligned using flexbox, see the image from the project requirement:

![figma2](https://user-images.githubusercontent.com/53124548/133517240-8819c73c-22e5-4ffa-8cc2-5aaba212ae80.png)

- [ ] You must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using  [the templates in Figma.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)